build_dep.sh

```diff
@@ -170,25 +175,15 @@ function addAlias {
 
 # fedora29:1, centos7.5:2, centos6.5:3, ubuntu18:4, ubuntu16:5
 function getSystemVer {
-    result=`cat /proc/version|grep fc`
-    if [[ -n $result ]]; then
-        return 1
-    fi
-    result=`cat /proc/version|grep el7`
-    if [[ -n $result ]]; then
-        return 2
-    fi
-    result=`cat /proc/version|grep el6`
-    if [[ -n $result ]]; then
-        return 3
-    fi
-    result=`cat /proc/version|grep ubuntu1~18`
-    if [[ -n $result ]]; then
-        return 4
+    if [[ -e /etc/redhat-release ]]; then
```

Comment:
why not check `/etc/os-release` to get os info?

Reply:
Centos6 does not have this file